build-in-container/scripts/build-mariner.sh

```diff
@@ -87,8 +87,11 @@ done
 source /mariner/scripts/setup.sh
 
 if [[ "${container_type}" == "build" ]]; then
-    # exit if SPECS/ is empty
-    if [ ! "$(ls -A $SPECS_DIR)" ]; then exit; fi
+    # exit if SPECS/ is empty && container_type is build as there is nothing to be done
+    if [ ! "$(ls -A $SPECS_DIR)" ]; then
+        echo -e "-------- \033[31m ALERT: Exiting the build container. No specs found in SPECS which is mounted at \$mariner_dir/SPECS on the host\033[0m --------"
```

Comment:
Looks like `$mariner_dir` is printed as is. Do we expect the user to know what `$mariner_dir` is? Or asked differently, can the direct path on the host be given to the user (contents of `$mariner_dir`)?

Reply:
Yes, this is intended. $mariner_dir is described in the README, and is to be provided by the user when running the tool. It doesn't have a meaning inside the container, therefore, the contents can't be printed from this echo unfortunately.

Reply:
https://github.com/microsoft/CBL-MarinerTutorials/tree/main/build-in-container#:~:text=Optional%20arguments%20%0A%0A%20%20%2D%2D-,mariner_dir,-directory%20to%20use

Reply:
Ah makes sense. Thanks for the explanation @neha170!